cmd/containerd-shim-runhcs-v1/exec_hcs.go

```diff
@@ -181,7 +192,7 @@ func (he *hcsExec) Status() *task.StateResponse {
 	}
 }
 
-func (he *hcsExec) Start(ctx context.Context) (err error) {
+func (he *hcsExec) startInternal(ctx context.Context, startContainer bool) (err error) {
```

Comment:
I don't think I really like the name of this boolean. Maybe you can explain a bit more about the scenarios that `startContainer` is true. I think it would be more clear if the option was called `startInitContainer` or something along those lines?

Reply:
I have changed it to `initializeContainer`.